commands/account/list.ts

```diff
@@ -115,7 +112,7 @@ export async function handler(
   logger.log(i18n(`${i18nKey}.configPath`, { configPath: configPath! }));
   logger.log(
     i18n(`${i18nKey}.defaultAccount`, {
-      account: getConfigDefaultAccount()!,
+      account: getConfigDefaultAccount().accountId,
```

Comment:
`getConfigDefaultAccount ` can throw. Are we protected from it throwing here because we make sure that it won't throw in our middleware?